boards/arm/nucleo_l496zg/Kconfig.defconfig

```diff
@@ -0,0 +1,63 @@
+# Kconfig - STM32L496ZG Nucleo board configuration
+#
+# Copyright (c) 2018 Ioannis Konstantelias <ikonstadel@gmail.com>
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+if BOARD_NUCLEO_L496ZG
+
+config BOARD
+	default "nucleo_l496zg"
+
+config UART_3
```

Comment:
This one should not be here, better under if SERIAL. But I think we should remove it completly

Reply:
I agree with removing `UART_3`.

Reply:
Seems you still need to remove it from here

Reply:
As I see the Kconfig.defconfig, it has been removed. `UART_3` is removed only from this file and not from pinmux.c. Is that you are talking about?